### PR TITLE
PW-3714 paypal cancel pop-up before it loads

### PR DESCRIPTION
**Pull request by zaiddreakh**
## Summary

When PayPal dialog is closed before the content is loaded payment page is blocked - order is created but not failed
https://github.com/Adyen/adyen-salesforce-commerce-cloud/issues/403

## Tested scenarios

**Fixed issue**:

### CODE REVIEW

File: src/cartridges/int_adyen_SFRA/cartridge/client/default/js/adyen_checkout/checkoutConfiguration.js [hunk 1]
```diff
@@ -39,21 +41,30 @@ function getPaypalConfig() {
       helpers.paymentFromComponent(state.data, component);
     },
     onCancel: (data, component) => {
+      window.paypalTerminatedEarly = false;
```
Comment (cyattilakiss) [inline src/cartridges/int_adyen_SFRA/cartridge/client/default/js/adyen_checkout/checkoutConfiguration.js:44]: why is this line needed in all the callbacks?
I guess it would be enough only once the properties are defined like the `showPayButton: true,`.

File: src/cartridges/int_adyen_controllers/cartridge/js/pages/checkout/adyen-checkout.js [hunk 2]
```diff
@@ -110,12 +122,15 @@ function initializeBillingEvents() {
           );
         },
         onCancel: (data, component) => {
+          paypalTerminatedEarly = false;
```
Comment (cyattilakiss) [inline src/cartridges/int_adyen_controllers/cartridge/js/pages/checkout/adyen-checkout.js:125]: same here, maybe I just don't get the logic behind it :)

Comment (pavelkukov) [reply src/cartridges/int_adyen_controllers/cartridge/js/pages/checkout/adyen-checkout.js:125]: Hi @cyattilakiss , a bit late to reply but...
Please, see the problem described in the original ticket.
https://github.com/Adyen/adyen-salesforce-commerce-cloud/issues/403

If the PayPal window is terminated by the user, there is no way to detect this in the checkout.

Best,
Pavel

Comment (zaiddreakh) [reply src/cartridges/int_adyen_controllers/cartridge/js/pages/checkout/adyen-checkout.js:125]: Hi @pavelkukov,

this was fixed here https://github.com/Adyen/adyen-salesforce-commerce-cloud/pull/411

Regards,
Zaid
Adyen

Comment (pavelkukov) [reply src/cartridges/int_adyen_controllers/cartridge/js/pages/checkout/adyen-checkout.js:125]: Thank you, @zaiddreakh !

Comment (pavelkukov) [reply src/cartridges/int_adyen_controllers/cartridge/js/pages/checkout/adyen-checkout.js:125]: @zaiddreakh , I see the fix in develop branch only. When will be released?
